README.md

```diff
@@ -1,34 +1,83 @@
+   page](https://github.com/EngFlow/auth/releases/latest).
 1. Copy the downloaded binary to a directory on the system `$PATH` and mark as
-   executable (if necessary)
-1. Configure `.bazelrc`: In the `.bazelrc` file of either your project or user,
-   add a `build` flag that sets `--credential_helper` for your cluster. For
+   executable (if necessary).
+1. In the `.bazelrc` file of either your project or user, add a line that sets `--credential_helper` for your cluster. For
    instance:
 
    ```
    build:engflow --credential_helper=example.cluster.engflow.com=/path/to/engflow_auth
    ```
 
    would configure the credential helper correctly when `--config=engflow` is
-   passed to a bazel invocation. See [Bazel's config
+   passed to a bazel invocation. You may remove existing `--tls_client_certificate`, `--tls_client_key`, and `--remote_header` flags for this cluster. See [Bazel's config
    documentation](https://bazel.build/run/bazelrc) for more info on bazelrc
    files, and [EngFlow setup
    documentation](https://docs.engflow.com/re/client/bazel-first-time.html#4-set-up-bazelrc)
-   for EngFlow-specific setup and tips.
+   for EngFlow-specific setup instructions.
+
+## Use
+
+1. Run `engflow_auth login [CLUSTER URL]` to obtain a credential. This prints a URL to visit in your browser.
+1. Visit the URL to complete the process, logging in if necessary. `engflow_auth` will download and store a credential in on your system's encrypted keyring.
+
+This process needs to be repeated after the credential expires, typically every 90 days.
+
+## Use in a non-interactive environment
+
+You can use `engflow_auth` to authenticate when no web browser is available, for example, on a continuous integration and testing server.
+
+1. You may wish to create a service account with your authentication provider, then log into your EngFlow cluster with that account. The credential created here will let Bazel authenticate as this account.
+1. On a machine with a web browser, complete the login process as described above:
+
+    ```bash
+    engflow_auth login [CLUSTER URL]
+    ```
+
+1. Export the credential to stdout using the command below:
+
+    ```bash
+    engflow_auth export [CLUSTER URL]
+    ```
+
+1. Save this credential as a secret, accessible in the non-interactive environment. For example, if you're using GitHub Actions, you can save this as a GitHub secret, then grant access in workflows that need it.
+1. At the beginning of a job, retrieve the secret and import it on stdin using the command below. The `-store=file` flag may be necessary to store the credential as an unencrypted file instead of your encrypted keyring. Non-interactive environments typically don't have an encrypted keyring.
+
+    ```bash
+    engflow_auth import -store=file <<<"${ENGFLOW_CRED}"
+    ```
+
+1. At the end of a job, remove the credential using the command below.
+
+    ```bash
+    engflow_auth logout [CLUSTER URL]
+    ```
+
```

Comment:
Maybe link to an example of how it is set up? https://github.com/EngFlow/auth/blob/main/.github/workflows/main.yml#L57

Reply:
Added links.